doc/source/visualizing/plots.rst

```diff
@@ -287,6 +287,9 @@ In this case, a normal vector for the cutting plane is supplied in the second
 argument. Optionally, a ``north_vector`` can be specified to fix the orientation
 of the image plane.
 
+.. note:: Not every data types have support for off-axis slices yet.
```

Comment:
> can you point to the idea of doing a thin off-axis projection for sph?

Good point


>  this is a really straightforward fix

I'm sure you mean "for SPH", but I really don't think that making off-axis ops work in non-cartesian coordinates would be easy, and that's why I deliberately phrased this as generally as I could. Am I mislead ?

Reply:
> can you point to the idea of doing a thin off-axis projection for sph?

(if it's indeed trivial to implement, maybe I actually don't need to document a workaround ? in any case I don't wish to work on this implementation myself for the time being, as easy as it may be)

Reply:
So should I just document the workaround for now ?

Reply:
Yup

Reply:
done !